pallets/staking-rewards/src/lib.rs

```diff
@@ -192,10 +200,17 @@ pub mod pallet {
                 .pool_balance
                 .saturating_add(T::Currency::minimum_balance());
             if T::Currency::free_balance(&account_id) < amount {
-                // Set the stakinig rewards pool account balance to the initial value.
+                // Set the staking rewards pool account balance to the initial value.
                 // Dropping the resulting imbalance as the funds are minted out of thin air.
                 let _ = T::Currency::make_free_balance_be(&account_id, amount);
             }
+
+            // create account for the rent pool
+            let _ = T::Currency::make_free_balance_be(
```

Comment:
btw doesnt deposit_creating better?

Reply:
frankly speaking I just used the same approach as for the staking rewards pool )